### DISCUSSION TITLE
Is it possible to create own subdomains in github pages?

### DISCUSSION THREAD

**Discussion by SAMALAMA37** [community/community#54144] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

For example: subdomain.example.github.io

**Comment by airtower-luna** [comment, marked as answer]
Simple answer: No. You get the `github.io` subdomain for your user/organization name automatically, the alternative is to use custom domain(s).

**Comment by AvverbioPronome** [comment]
No, as that would be a real mess for both DNS and TLS certificate Subject Alternative Names. 

Both have wildcards, but wildcards do not apply to multiple labels. The current certificate is valid for `*.github.io`, meaning that it is valid for `example.github.io` but not valid for `hello.example.github.io`
